### PR TITLE
New structure for samples

### PR DESCRIPTION
**Pull request by ppamorim**
Hi everyone.
I've just fixed some issues and downgraded the API to 14 for every sample. 
Be free to review these codes!

### CODE REVIEW

File: rebound-android-playground/src/main/res/layout/activity_playground.xml [hunk 1]
```diff
@@ -1,37 +1,30 @@
...
+      android:layout_gravity="center"
+      android:padding="40dp"/>
+
     <ListView
       android:id="@+id/list_view"
       android:layout_width="match_parent"
       android:layout_height="match_parent"
-      android:layout_weight="1"
-      />
+      android:background="#ccc"/>
```
Comment (mutexkid) [inline rebound-android-playground/src/main/res/layout/activity_playground.xml]: you may want to consider extracting this color value to the colors.xml file (to be consistent with the rest of the file, for example line 6)

Comment (mutexkid) [inline rebound-android-playground/src/main/res/layout/activity_playground.xml]: you may want to consider extracting this color value to the values/colors.xml file

File: rebound-android-playground/src/main/java/com/facebook/rebound/playground/app/PlaygroundActivity.java [hunk 2]
```diff
@@ -140,62 +144,54 @@ public void onEnd() {
...
   private class ExampleListAdapter implements ListAdapter {
 
-    @Override
-    public void registerDataSetObserver(DataSetObserver observer) {}
+    @Override public void registerDataSetObserver(DataSetObserver observer) { }
 
-    @Override
-    public void unregisterDataSetObserver(DataSetObserver observer) {
-    }
+    @Override  public void unregisterDataSetObserver(DataSetObserver observer) { }
```
Comment (mutexkid) [inline rebound-android-playground/src/main/java/com/facebook/rebound/playground/app/PlaygroundActivity.java]: formatting looks slightly off here

File: rebound-android-playground/src/main/java/com/facebook/rebound/playground/examples/BallExample.java [hunk 3]
```diff
@@ -70,75 +80,48 @@ public BallExample(Context context, AttributeSet attrs) {
...
-            points.add(new PointF(j, i));
-          }
-        }
+  @Override protected void onSizeChanged(int width, int height, int oldWidth, int oldHeight) {
+    super.onSizeChanged(width, height, oldWidth, oldHeight);
+    centerX = getWidth() / 2f;
+    centerY = getHeight() / 2f;
+    xSpring.setCurrentValue(centerX).setAtRest();
+    ySpring.setCurrentValue(centerY).setAtRest();
+    int offsetH = (int) ((getHeight() - (2 * radius)) % 800) / 2;
```
Comment (mutexkid) [inline rebound-android-playground/src/main/java/com/facebook/rebound/playground/examples/BallExample.java]: should the magic number 800 be defined as some sort of constant?

File: rebound-android-playground/src/main/java/com/facebook/rebound/playground/examples/BallExample.java [hunk 4]
```diff
@@ -70,75 +80,48 @@ public BallExample(Context context, AttributeSet attrs) {
...
+    int offsetW = (int) ((getWidth() - (2 * radius))  % 800) / 2;
+    for (float i = offsetH + radius; i < getHeight() - offsetH; i += 400) {
```
Comment (mutexkid) [inline rebound-android-playground/src/main/java/com/facebook/rebound/playground/examples/BallExample.java]: same thing with 400 - should this be a constant of some sort?

Comment (ppamorim) [inline rebound-android-playground/src/main/java/com/facebook/rebound/playground/examples/BallExample.java]: Can be, it's from the code of the master branch (by facebook). I don't have done any change there.

File: rebound-android-playground/src/main/java/com/facebook/rebound/playground/examples/BallExample.java (hunk 3 above)
Comment (ppamorim) [inline rebound-android-playground/src/main/java/com/facebook/rebound/playground/examples/BallExample.java]: Can be, it's from the code of the master branch (by facebook). I don't have done any change there.

**Comment by ppamorim** [comment]
@willbailey Any news?

**Comment by willbailey** [comment]
- I don't see a reason to pull in butterknife. I want to keep this code simple and minimize dependencies.
- Some of the stylistic changes e.g. annotations on the same line as the method declaration are inconsistent with the rest of the codebase.
- If you want to downgrade the API level to 14, let's make that a separate pull request.

**Comment by ppamorim** [comment]
@willbailey ButterKnife is only implemented at sample project and I'm using the square code style. 
Ok... But why? 😱

**Comment by willbailey** [comment]
No reason to add the cognitive load of learning the butter knife api. The point of these examples is to be as minimal as possible while showing the concept in rebound.

For code style, it's a matter of opinion/preference, but I think consistency in a given codebase is important. The style in this codebase so far has been annotations on their own line, so let's keep it consistent.

**Comment by ppamorim** [comment]
I understand your point, but, can reduce verbosity raise the cognitive load? If yes, ok, please stop the bus... I need go out.

**Comment by willbailey** [comment]
Reducing verbosity through an API the user doesn't know can increase the cognitive load because you are introducing a previously unknown capability to the environment. e.g. what are these @Bind annotations? Is that part of Rebound? If I'm new to the library I don't know.

**Comment by ppamorim** [comment]
You're correct with this point and I don't have mind about this problem. We can simplify the source and create a more complete wiki to help new developers.
;)

**Comment by ppamorim** [comment]
@willbailey I changed some items that you noticed to me. Now I have some questions:
Do you know any way to remove the getViewTreeObserver() in PhotoGallerySample and ExampleContainerView classes? I tried to use onFinishInflate() method at these classes but the effect doesn't work correctly. Can you review?

**Pull request closed without merging**